python-lib/dku_io_utils/utils.py

```diff
@@ -80,7 +80,7 @@ def write_to_folder(object_to_save, folder, path, object_type):
             )
 
 
-def set_column_description(output_dataset, column_description_dict, input_dataset=None, to_lowercase=False):
+def set_column_description(output_dataset, column_description_dict, input_dataset=None):
```

Comment:
Do we need to keep the new version?

Reply:
because of the create_evaluation_results_columns_descriptions method, we don't need to modify the set_column_description that you coded in another plugin